airbrake/handler.py

```diff
@@ -25,7 +25,7 @@ class AirbrakeHandler(logging.Handler):
     # pylint: disable=too-many-arguments
 
     def __init__(self, airbrake=None, level=logging.ERROR, project_id=None,
-                 api_key=None, environment=None, base_url=None, hostname=None):
+                 api_key=None, **config):
```

Comment:
Does the airbrake api call this "context" ? I haven't looked at the api docs in a while, but I see that in the payload it is the value for `"context"`.